Wikipedia/Code/WMFTwoFactorPasswordViewController.swift

```diff
@@ -17,18 +19,51 @@ class WMFTwoFactorPasswordViewController: WMFScrollViewController, UITextFieldDe
     public var captchaID:String?
     public var captchaWord:String?
     
+    func backupOathTokenToggleTapped(_ recognizer: UITapGestureRecognizer) {
+        guard recognizer.state == .ended else {
+            return
+        }
+        useBackupOathToken = !useBackupOathToken
+    }
+    
+    fileprivate var useBackupOathToken: Bool = false {
```

Comment:
Oh, it's Oath. I asked Gergo about the difference a couple weeks ago but I don't recall how he explained it. Check out the url parameters in the example at the very bottom of this page: 
https://www.mediawiki.org/wiki/API:Login#The_clientlogin_action